_posts/2018-04-17-making-a-components-pod.markdown

```diff
@@ -485,19 +503,27 @@ Then run `bundle install`. Next we need to add GitDawg, and our custom Podspecs
 ```diff
 source 'https://github.com/CocoaPods/Specs.git'
 
-[...]
+# Local Pods w/ custom changes
+pod 'SwipeCellKit', :path => 'Local Pods/SwipeCellKit'
+pod 'GitHubAPI', :path => 'Local Pods/GitHubAPI'
+pod 'GitHubSession', :path => 'Local Pods/GitHubSession'
 
++ # React Native Support
++ plugin 'cocoapods-fix-react-native'
```

Comment:
This seems to have fixed my various `RCT*.h not found` issues. Going to try and break again to confirm.

Reply:
<img width="235" alt="screen shot 2018-04-18 at 11 54 55" src="https://user-images.githubusercontent.com/1851976/38928026-759a8e94-42ff-11e8-8e67-122937910aa6.png">